radicle-httpd/src/api/v1/projects.rs

```diff
@@ -12,16 +12,15 @@ use serde::{Deserialize, Serialize};
 use serde_json::json;
 use tower_http::set_header::SetResponseHeaderLayer;
 
-use radicle::cob::issue::{Action, Issues};
-use radicle::cob::patch::Patches;
 use radicle::cob::{thread, ActorId, Tag};
 use radicle::identity::Id;
 use radicle::node::NodeId;
 use radicle::storage::git::paths;
-use radicle::storage::{ReadRepository, ReadStorage};
+use radicle::storage::{ReadRepository, ReadStorage, WriteRepository};
 use radicle_surf::{Glob, Oid, Repository};
 
 use crate::api::error::Error;
+use crate::api::project::cob::{issue, patch};
```

Comment:
Hmm why not import directly through `radicle::cob::` instead of re-exporting from project?

Reply:
yeah that was my first version, but according to https://github.com/radicle-dev/heartwood/pull/377#pullrequestreview-1322265433 I changed it to use `project` where I needed to make `radicle::cob` public to be able to use it.
Will change it back to use `radicle::cob` directly.
Not sure what is the heartwood idiomatic way of using the own modules yet.